definitions/npm/webpack_v4.x.x/flow_v0.104.x-/webpack_v4.x.x.js

```diff
@@ -569,6 +569,14 @@ declare module 'webpack' {
       | false,
     entry?: Entry,
     externals?: Externals,
+    infrastructureLogging?: {
+      level?: 'none' | 'error' | 'warn' | 'info' | 'log' | 'verbose',
+      debug?:
+        | string
+        | RegExp
+        | ((string) => boolean)
+        | Array<string | RegExp | ((string) => boolean)>,
+    },
```

Comment:
**suggestion:** should be either exact or inexact.

Reply:
Aye aye 👍 !